frontend/vite.config.ts

```diff
@@ -5,5 +5,13 @@ export default defineConfig({
   plugins: [sveltekit()],
   test: {
     include: ['src/**/*.{test,spec}.{js,ts}']
+  },
+  server: {
+    host: '0.0.0.0',
+    hmr: {
+        host: 'localhost'
+    },
```

Comment:
```suggestion
```

Let's instead put this in the `package.json` so that different environments can have different settings. At the same time let's change the port to 3000...mostly because I can never remember 5173. Change "dev": "vite dev",` to `"dev": "vite dev --host 0.0.0.0 --port 3000",`